src/bridge/Bridge.js

```diff
@@ -234,6 +236,12 @@ class Bridge extends React.Component {
     this.setState({ txnHashCursor })
   }
 
+  setTxnConfirmations(txnHash, txnConfirmations) {
+    this.setState((prevState, _) =>
+                  ({txnConfirmations: { ...prevState.txnConfirmations,
+                                        [txnHash]: txnConfirmations}}))
+  }
+
```

Comment:
`StatelessTransaction` needs to communicate state change (confirmations going up) to `SentTransaction`, which is not possible through `pushRoute`, since it only ever gets called once in the `submit` method. Only the `render` method on a component gets called on state updates.